app/models/financialDetails/FinancialDetailsResponseModel.scala

```diff
@@ -115,17 +115,18 @@ case class FinancialDetailsModel(balanceDetails: BalanceDetails,
 
   def isAllInterestPaid()(implicit user: MtdItUser[_]): Boolean = documentDetails.forall(_.interestIsPaid)
 
-  def validChargeTypeCondition: String => Boolean = documentDescription => {
-    documentDescription == "ITSA- POA 1" ||
-      documentDescription == "ITSA - POA 2" ||
-      documentDescription == "TRM New Charge" ||
-      documentDescription == "TRM Amend Charge"
+  def validChargeTypeCondition: DocumentDetail => Boolean = documentDetail => {
```

Comment:
might be a good idea to add unit test around this method in FinancialDetailsResponseModelSpec;

Reply:
Have done now